### ISSUE TITLE
Support for namespace

### ISSUE THREAD

**Issue by lelabo-marc** [isaacs/github#1486]
It would be great if github support namespace inside organisation or profiles.
I would like to be able to group github projects by THEME / PRODUCT / ANYTHING that make sense.

**Comment by cirosantilli** [comment]
Possible duplicate of: https://github.com/isaacs/github/issues/49 | https://github.com/isaacs/github/issues/136

**Comment by lelabo-marc** [comment]
Indeed, I searched for issues with namespace as keyword and tried to find any others that would match.

Its pretty much the same as #49 .

Issue #136 is more complementary things that would be nice to have in addition to this feature.
